architecture/core_concepts/routes.adoc

```diff
@@ -683,6 +689,11 @@ validate the endpoint certificate, securing the connection from the router to
 the destination. This field is required, but only for re-encryption.
 ====
 
+[NOTE]
+====
+Reencrypt routes can have an insecureEdgeTerminationPolicy the same as edge-terminated routes.
```

Comment:
Reencrypt routes can have an insecureEdgeTerminationPolicy with all of the same values as for edge-terminated routes.